src/picker.js

```diff
@@ -281,6 +282,10 @@ angular.module("ion-datetime-picker", ["ionic"])
                     changeViewData();
                 };
 
+                $scope.timeFieldType = function() {
+                  return $scope.telTypeForTimeFields ? "tel" : "text";
```

Comment:
In the future, please try to maintain the existing code style, including indentation style, even if you think that it's weird and unusual :)